src/target/riscv/riscv-013.c

```diff
@@ -857,9 +859,11 @@ static int dm013_select_target(struct target *target)
 	return dm013_select_hart(target, info->index);
 }
 
-static int execute_abstract_command(struct target *target, uint32_t command)
+static int execute_abstract_command(struct target *target, uint32_t command,
+		unsigned int *p_abstractcs_err)
```

Comment:
Can this be called just `cmderr`?

```suggestion
		unsigned int *cmderr)
```

Reply:
addressed. replaced with `abstractcs_err`